src/MarginTrading.AzureRepositories/MarginTradingOrdersHistoryRepository.cs

```diff
@@ -78,6 +78,12 @@ public class MarginTradingOrderHistoryEntity : TableEntity, IOrderHistory
         public List<MatchedOrder> MatchedCloseOrders { get; set; } = new List<MatchedOrder>();
         decimal IOrderHistory.SwapCommission => (decimal) SwapCommission;
         public double SwapCommission { get; set; }
+        
+        public string EquivalentAsset { get; set; }
+        decimal IOrderHistory.OpenPriceEquivalent => (decimal) OpenPriceEquivalent;
+        public double OpenPriceEquivalent { get; set; }
+        decimal IOrderHistory.ClosePriceEquivalent => (decimal) ClosePriceEquivalent;
+        public double ClosePriceEquivalent { get; set; }
```

Comment:
Are you sure we don't need to add any other price here, ex. ExpectedOpenPrice?

Reply:
Will be checked